spec/controllers/concerns/forced_reauthentication_concern_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe ForcedReauthenticationConcern do
+  class TestClass
+    include ForcedReauthenticationConcern
+
+    attr_reader :session
+
+    def initialize(session = {})
+      @session = session
+    end
+  end
```

Comment:
can we rewrite this as a `let`? Otherwise `TestClass` leaks into the global namespace

```suggestion
  let(:test_class) do
    Class.new do
      include ForcedReauthenticationConcern

      attr_reader :session

      def initialize(session = {})
        @session = session
      end
    end
  end
  let(:instance) { test_class.new }
```